### DISCUSSION TITLE
failed to pull and unpack image "docker.io/splunk/splunk-operator:2.3

### DISCUSSION THREAD

**Discussion by Zeroxcharisma** [splunk/splunk-operator#1187] [Q&A] [answered]
why do i keep receiving this error ! even tho in the chart is well written as 2.3.0

**Comment by vivekr-splunk** [comment, marked as answer]
Hello @Zeroxcharisma , thank you for finding this issue. We have fixed this issue in PR https://github.com/splunk/splunk-operator/pull/1190 . should be updated soon
